docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+### YAML Format
+
+Rasa Open Source 2.0 uses [YAML](https://yaml.org/spec/1.2/spec.html) as
+a unified and extendable way to manage all training data,
+including NLU data, stories and rules. 
+
+With the YAML format, training data can be split over any number of YAML files,
+and every file can contain any kind of data. 
+The training data parser will read the top level keys in each file to decide
+what kind of data is in a section at training time.
+
+The [domain](../glossary#domain) uses
+the same YAML format as the training data and can also be split across
+multiple files or combined with training data in one file. The domain includes
+the definitions for [responses](../core/domains#responses) and [forms](../core/forms).
+See the [documentation for domains](../core/domains) to learn more.
+
+### High Level Structure
+
+Each file can contain one or more **keys** with corresponding training
+data. One file can contain multiple keys, as long as there is not more
+than one of a certain key in a single file. The available keys are:
+
+- `version`
+- `stories`
+- `nlu`
+- `rules`
+- `e2e_tests`
+
+All YAML training data files should specify the `version` key in order
+to be parsed correctly. Training data files with no `version` specified will be assumed to
+be in the format of the latest version of Rasa Open Source. Training data files with a version greater than is currently
+available for Rasa Open Source will be skipped.
+
+### Example
+
+Here's a short example which keeps all training data in a single file:
+
+```yaml
+version: "2.0"
+
+stories:
+- story: greet and faq
+  steps:
+  - intent: greet
+  - action: utter_greet
+  - intent: faq
+  - action: respond_faq
+
+rules:
+- rule: Greet user
+  steps:
+  - intent: greet
+  - action: utter_greet
+
+responses:
+  faq/language:
+  - text: |
```

Comment:
I think we should not use blocks for these but rather use simple strings. e.g.
`- text: "I can only do English at the moment"`

Reply:
it avoids people confusing this with the nlu syntax and doing odd things like adding a `-` in the beginning

Reply:
Ah right, I should actually remove the responses example entirely because at the top it says they're part of the domain